cheatsheets/associations.cheatmd

```diff
@@ -0,0 +1,272 @@
+# Associations
+
+## Has many / belongs to
+{: .col-2}
+
+### The has many association
+
+```elixir
+defmodule Movie do
+  use Ecto.Schema
+
+  schema "movies" do
+    field :title, :string
+    field :release_date, :date
+    has_many :characters, Character
+  end
+end
+```
+
+### The belongs to association
+
+```elixir
+defmodule Character do
+  use Ecto.Schema
+
+  schema "characters" do
+    field :name, :string
+    field :age, :integer
+    belongs_to :movie, Movie
+  end
+end
+```
+
+## Has one / belongs to
+{: .col-2}
+
+### The has one association
+
+```elixir
+defmodule Movie do
+  use Ecto.Schema
+
+  schema "movies" do
+    field :title, :string
+    field :release_date, :date
+    has_one :distributor, Distributor
+  end
+end
+```
+
+### The belongs association
+
+```elixir
+defmodule Distributor do
+  use Ecto.Schema
+
+  schema "distributors" do
+    field :name, :string
+    belongs_to :movie, Movie
+  end
+end
+```
```

Comment:
```suggestion
```elixir
defmodule Movie do
  use Ecto.Schema

  schema "movies" do
    field :title, :string
    field :release_date, :date
    has_one :screenplay, Screenplay
  end
end
```

### The belongs association

```elixir
defmodule Screenplay do
  use Ecto.Schema

  schema "screenplays" do
    field :lead_writer, :string
    belongs_to :movie, Movie
  end
end
```
```

Not sure how accurate you want to be. I was a bit thrown off by the distributor association because for sure they are their own independent company not belonging to a single movie.

Reply:
Thanks, I'll change that.